MH4U Dex/Images.xcassets/Region Icons/ancestral_steppe-icon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x",
+      "filename" : "Image.png"
```

Comment:
A big issue will be the monster icons, though.  I don't think I can get away with leaving those out.  I'll have to figure something out.